app/Services/TransactionSyncService.php

```
@@ -0,0 +1,177 @@
+
+        return $stats;
+    }
+
+    /**
+     * Calculate date range for sync.
+     *
+     * @param Account $account
+     * @param int $maxDays
+     * @return array
+     */
+    public function calculateDateRange(Account $account, int $maxDays = 90, bool $forceMax = false): array
```

PHPStan (2.1.15)

Method App\Services\TransactionSyncService::calculateDateRange() return type has no value type specified in iterable type array.
See: https://phpstan.org/blog/solving-phpstan-no-value-type-specified-in-iterable-type

(missingType.iterableValue)